pyproject.toml

```diff
@@ -12,11 +12,12 @@ license = "Apache 2.0"
 [tool.poetry.dependencies]
 python = "<3.11,>=3.7.1"
 requests = "^2.25.1"
-singer-sdk = "^0.19.0"
+# singer-sdk = "^0.19.0"
+singer-sdk = {git = "https://github.com/meltano/sdk.git", rev = "main", extras=["testing"]}
```

Comment:
eek, the SDK doesn't have testing released yet? Going to come back to this

Reply:
I think leaving this for now in the PR is fine to do some testing with it, but we should wait for the SDK to be updated to roll this out.